docs/knowledge_base/tips/colab_with_ngrok/index.md

```diff
@@ -0,0 +1,184 @@
+# Sharing a Taipy Application on Colab with Ngrok
+
+This article will show you how to put a Taipy Application on Colab, which is a Google Notebook 
+platform. You'll also learn how to make it accessible through a public URL created with Ngrok.
+
+![Monthly Production Planning](Sharing_Taipy_Ngrok_1.png){width=80%}
+
+## Problem statement
+
+For people who want to create strong graphical interfaces without needing to be an expert in GUI 
+(Graphical User Interface), Taipy is definitely the solution.
+
+In addition to coding with Python scripts, Taipy works seamlessly in 
+[Notebook environments](https://www.taipy.io/tips/taipy-gui/taipy-in-jupyter-notebooks/).
+You can use Notebooks on your computer and on platforms like:
+
+- [Google Colab](https://colab.google/)
+- [Kaggle](https://www.kaggle.com/)
+- [Databricks](https://www.databricks.com/)
+- [Amazon SageMaker](https://aws.amazon.com/fr/sagemaker/)
+- [IBM Watson Studio](https://www.ibm.com/products/watson-studio)
+
+In this article, we will demonstrate how to put a Taipy application on Notebooks that are hosted 
+on the Google Colab platform. We will achieve this by creating a public URL using 
+[Ngrok](https://ngrok.com/).
+
+![Problem statement](Sharing_Taipy_Ngrok_1.png){width=80%}
+
+To begin, let's briefly explain what Ngrok and Google Colab are.
+
+## What is Google Colab?
+
+[Google Colab](https://colab.google/) is a preferred tool for Data Scientists. It offers an 
+advanced Notebook environment that provides access to powerful hardware, 
+including CPUs and GPUs, essential for tasks like deep learning, without the need to buy 
+physical machines.
+
+However, Google Colab is cloud-based, so you can't directly see the graphical user interface 
+(GUI) of your web application. But don't worry, we'll show you how Ngrok can help solve this issue.
+
+## What is Ngrok?
+
+Ngrok enables you to share your application directly on the internet. It sets up a connection 
+from your computer to the outside world through a public URL.
+
+Please note that the URL is temporary when using the free version of Ngrok, but you can make it 
+permanent by opting for a paid solution.
+
+## Setting up your Taipy application on Google Colab
+
+Now, let's begin:
+
+For our example, we will use a Sentiment Analysis Application from 
+[Taipy's Getting Started](https://github.com/Avaiga/demo-sentiment-analysis) guide. This 
```

Comment:
This is not a getting started anymore. We should change the name.

Reply:
Changed to "Demo sentiment analysis"